.gitignore

```diff
@@ -6,3 +6,6 @@
 /static/
 /media/
 gettogether.code-workspace
+.vscode/
+venv/
```

Comment:
I think the virtualenv is located in the `env/` folder, so you shouldn't need to have `venv`.